src/gtk/webview_webkit2.cpp

```diff
@@ -536,14 +542,14 @@ bool wxWebViewWebKit::Create(wxWindow *parent,
 
     PostCreation(size);
 
-    /* Open a webpage */
-    webkit_web_view_load_uri(m_web_view, url.utf8_str());
-
-    // last to avoid getting signal too early
     g_signal_connect_after(m_web_view, "load-changed",
                            G_CALLBACK(wxgtk_webview_webkit_load_changed),
                            this);
 
+    LoadURL(url);
+
+    m_runScriptCount = 0;
```

Comment:
This should be really done in the base class ctor once instead of doing it in all the different backends.